sklearn/cross_validation.py

```diff
@@ -839,67 +847,11 @@ def __len__(self):
         return self.n_iter
 
 
-def _validate_shuffle_split(n, test_size, train_size):
-    if test_size is None and train_size is None:
-        raise ValueError(
-            'test_size and train_size can not both be None')
-
-    if test_size is not None:
-        if np.asarray(test_size).dtype.kind == 'f':
```

Comment:
What's wrong with doing `isinstance(test_size, float)` and similarly int which is in the present `_split.py`?